core/src/main/resources/jenkins/model/Jenkins/help-rawBuildsDir.html

```diff
@@ -1,15 +1,28 @@
 <div>
-    Specify where Jenkins would store records of the past builds.
-    This value can include the following variables.
-
+    Specifies where Jenkins will store build records on the file system. This
+    includes the console output and other metadata generated by a build.
+    <p>
+    This value may include the following variables:
     <ul>
-        <li><tt>${JENKINS_HOME}</tt> &mdash; Jenkins home directory.
-        <li><tt>${ITEM_ROOTDIR}</tt> &mdash; Root directory of a job for which the default workspace is allocated.
-        <li><tt>${ITEM_FULL_NAME}</tt> &mdash; '/'-separated job name, like "foo/bar".
+        <li><tt>${JENKINS_HOME}</tt> &mdash; Absolute path of the Jenkins home
```

Comment:
like said elsewhere, [tt](https://developer.mozilla.org/en/docs/Web/HTML/Element/tt) is obsolete. Prefer using `code`